### ISSUE TITLE
Document how to pass arguments along to lando/ddev

### ISSUE THREAD

**Issue by dalin-** [fourkitchens/fire#23]
I often type `drush cex -y`.  With FIRE that'd be `fire drush cex -y`.  But...
<img width="276" alt="Screenshot 2024-03-07 at 1 58 03 PM" src="https://github.com/fourkitchens/fire/assets/1127349/a5d81f52-0cec-45d0-b66c-b83a1c836fa9">
Can we pass unknown arguments along to lando/ddev, similar to the way that Lando works?

This is related to, but different from
https://github.com/fourkitchens/fire/issues/21

**Comment by rigoucr** [comment]
Yes , this is already supported , but to do that have to use double hyphen, like when we use terminus remote commands 
example:  `fire drush  cex -- -y`

**Comment by dalin-** [comment]
Cool.  So then lets repurpose this issue to be about documenting that.
